### PR TITLE
1.12 Release Notes

### PR DESCRIPTION
**Pull request by anithamuthu79**
https://jira.mesosphere.com/browse/DCOS-42375

## Urgency
- [x] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by anithamuthu79** [comment]
@yishan-lin @orsenthil and @pavisandhu 1.12 RN PR is ready for review.

File: pages/1.12/release-notes/1.12/index.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+- Include base tech version in the services page.
+- Include node type (Public/Private) in the nodes page.
+- Remove CPU/MEM/Disk graphic from the nodes page.
+- Add the Mesosphere DC/OS logo above the UI.
+- Fix 100+ bugs and performance improvements.
+
+## Breaking Changes
+- DCOS_OSS-2256 - Remove the DC/OS web installer. 
+- DCOS_OSS-3714 - Replace `dcos-metrics` with Telegraf.
+- DCOS_OSS-4243 - By default, Marathon declines offers for agents in [maintenance mode](https://github.com/mesosphere/marathon/blob/master/changelog.md#maintenance-mode-support-production-ready-now-default). Requests to Marathon's events API (/v2/events when queried directly) for standby instances is no longer proxy, but instead responds with a redirect. Clients consuming Marathon's events API should be updated to follow redirects. Component such as updated to versions that follow redirects; for example, Marathon-LB should be updated to at least version v1.12.3. [See more details](https://github.com/mesosphere/marathon/blob/master/changelog.md#non-leaderstandby-marathon-instances-respond-to-v2events-with-a-redirect-rather-than-proxy).
```
Comment (timcharper) [inline pages/1.12/release-notes/1.12/index.md]: 👏

**Comment by anithamuthu79** [comment]
@yishan-lin  I've addressed your comments and I'm waiting for the **patching path matrix content** from @gauripowale. I've provided a link (https://docs.mesosphere.com/1.12/installing/production/patching/#supported-patching-path-matrix) in this PR to reflect the upcoming change on patching page.

File: pages/1.12/release-notes/1.12/index.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+- Mesos framework metrics are now [available](http://mesos.apache.org/documentation/latest/monitoring/#frameworks).
+- No longer require modifications when collecting metrics via Prometheus endpoint in 1.11.
+
+[enterprise]
+### Private Package Registry
+[/enterprise]
+- Enabled [on-premise package distribution and management](https://docs.mesosphere.com/1.12/administering-clusters/repo/package-registry/).
+- Enabled air-gapped Virtual Private Cloud package management.
+- Simplifies package artifact management.
+- Support for NFS and S3 as storage backends for package storage.
```
Comment (takirala) [inline pages/1.12/release-notes/1.12/index.md]: We don't support NFS Storage backend. @anithamuthu79 can you guide me on how this information is curated?

Comment (anithamuthu79) [reply pages/1.12/release-notes/1.12/index.md]: @takirala Pls refer the gDoc that @yishan-lin had shared https://docs.google.com/document/d/1xv-sq_mwG96GnvmbxQxiULPkvO5aVRKFqny-3hHpnN4/edit#heading=h.1w8qtj5xgzg.

Comment (takirala) [reply pages/1.12/release-notes/1.12/index.md]: Thanks for the links!

@somikbehera just as fyi, we don't support NFS storage.
@anithamuthu79  please remove the NFS as a supported storage.

Comment (anithamuthu79) [reply pages/1.12/release-notes/1.12/index.md]: Done. Ty @takirala.

**Pull request merged**